extensions/wikia/ArticleHeader/Hooks.class.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace ArticleHeader;
+
+class Hooks {
+    /**
+     * @param \OutputPage $out
+     * @param \Skin $skin
+     *
+     * @return bool
+     */
+    public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$out". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$skin". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)